src/components/Homepage/HomePage.test.js

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import HomePage from "./HomePage";
+import "@testing-library/jest-dom";
+
+jest.mock("axios");
+
+const MockHomePage = () => {
+  return (
+    <MemoryRouter>
+      <HomePage />
+    </MemoryRouter>
+  );
+};
+
+describe("Homepage", () => {
+  let response;
+  beforeEach(() => {
+    response = {
```

Comment:
What's the point of adding the response variable to beforeEach?

Reply:
fixed